Makefile

```diff
@@ -95,47 +95,47 @@ lint-cur:
 
 lint: ##@tests Run meta linter on code
```

Comment:
I wonder if we can replace all these lines with:
```
gometalinter $(LINT_EXCLUDE) --disable-all --enable=vet --enabled=gofmt --enable=goimports [--enable=...] cmd/... geth/...
```

And I guess this `linter_exclude_list.txt` can be removed as we can use `// nolint: golint` for individual functions?

Reply:
Yes, we could call gometalinter as you offered, but for me it's less convinient to use. 
About linter_exclude_list.txt i fully agree with you. I had to switch to #383 and push as is.

Reply:
I  think i could fix it after #383 and #375

Reply:
done